woocommerce-services.php

```diff
@@ -390,6 +390,66 @@ public function pre_wc_init() {
 			$this->taxjar->init();
 		}
 
+		public function get_service_schema_defaults( $schema ) {
+			$defaults = array();
+
+			if ( ! property_exists( $schema, 'properties' ) ) {
+				return $defaults;
+			}
+
+			foreach ( get_object_vars( $schema->properties ) as $prop_id => $prop_schema ) {
+				if ( property_exists( $prop_schema, 'default' ) ) {
+					$defaults[ $prop_id ] = $prop_schema->default;
+				}
+
+				if (
+					property_exists( $prop_schema, 'type' ) &&
+					'object' === $prop_schema->type
+				) {
+					$defaults[ $prop_id ] = $this->get_service_schema_defaults( $prop_schema );
+				}
+			}
+
+			return $defaults;
+		}
+
+		public function init_core_wizard_shipping_config() {
+			$store_country = WC()->countries->get_base_country();
+
+			if ( 'US' === $store_country ) {
+				$country_method = 'usps';
+			} elseif ( 'CA' === $store_country ) {
+				$country_method = 'canada_post';
+			} else {
+				return; // Only set up live rates for US and CA
+			}
+			
+			$zones_to_setup = array(
+				'domestic' => 1, // TODO: this might be too fragile/assumptive
+				'intl'     => 0,
+			);
+			
+			foreach ( $zones_to_setup as $option => $zone_id ) {
+				$option_name = "woocommerce_setup_{$option}_live_rates_zone";
+
+				if ( get_option( $option_name ) ) {
+					$method      = $this->get_service_schemas_store()->get_service_schema_by_id( $country_method );
+					$zone        = WC_Shipping_Zones::get_zone( $zone_id );
```

Comment:
In the same line, to make this more robust, you should check that `$zone !== false` and bail.